src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlCommand/SqlCommandCancelTest.cs

```diff
@@ -212,6 +212,37 @@ public static void AsyncCancelDoesNotWaitNP()
             AsyncCancelDoesNotWait(np_connStr).Wait();
         }
 
+        [CheckConnStrSetupFact]
+        public static void TCPAttentionPacketTest()
```

Comment:
Does this test cover [Issue#542](https://github.com/dotnet/SqlClient/issues/542) where SqlDependency is called?

Reply:
This test will not cover that. I didn't write reflection based test since the issue is regarding stored procedure availability in server, who's details are internal to driver. This test also captures the introduced problem.